From yaniv.rosner@broadcom.com  Thu Jun 23 16:09:53 2011
From: Yaniv Rosner <yaniv.rosner@broadcom.com>
Date: Wed, 1 Sep 2010 09:51:35 +0000
Subject: [PATCH 093/151] bnx2x: Add BCM84823 to the supported PHYs
Git-commit: 9a8a97e88ea893522779aecbc0ff3d7361f49c12
Patch-mainline: v2.6.37-rc1
References: FATE#311458 FATE#311459 FATE#311460 FATE#311461 FATE#311462 bnc#698050

Upstream commit: 9a8a97e88ea893522779aecbc0ff3d7361f49c12

Add BCM84823 to the supported PHYs

Signed-off-by: Yaniv Rosner <yanivr@broadcom.com>
Signed-off-by: Eilon Greenstein <eilong@broadcom.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Jeff Mahoney <jeffm@suse.com>
---
 drivers/net/bnx2x/bnx2x_main.c |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/net/bnx2x/bnx2x_main.c b/drivers/net/bnx2x/bnx2x_main.c
index 9ce61d3..513f04a 100644
--- a/drivers/net/bnx2x/bnx2x_main.c
+++ b/drivers/net/bnx2x/bnx2x_main.c
@@ -6227,7 +6227,8 @@ static void __devinit bnx2x_link_settings_supported(struct bnx2x *bp,
 			break;
 
 		case PORT_HW_CFG_XGXS_EXT_PHY_TYPE_BCM8481:
-			BNX2X_DEV_INFO("ext_phy_type 0x%x (BCM8481)\n",
+		case PORT_HW_CFG_XGXS_EXT_PHY_TYPE_BCM84823:
+			BNX2X_DEV_INFO("ext_phy_type 0x%x (BCM848xx)\n",
 				       ext_phy_type);
 
 			bp->port.supported |= (SUPPORTED_10baseT_Half |
-- 
1.6.4.GIT

